grails-app/services/org/pih/warehouse/data/LocationDataService.groovy

```diff
@@ -134,7 +135,9 @@ class LocationDataService {
         location.parentLocation = params.parentLocation ? Location.findByNameOrLocationNumber(params.parentLocation, params.parentLocation) : null
         location.organization = params.organization ? Organization.findByCodeOrName(params.organization, params.organization) : null
         location.address = createOrUpdateAddress(params, location?.address?.id)
-
+        if (!location.inventory) {
```

Comment:
One thing @jmiranda suggested, it should check if the location have no parent
`!location.inventory && !location.parentLocation`